BUILDING.md

```diff
@@ -78,30 +78,69 @@ echo 'export PKG_CONFIG_PATH="/usr/local/opt/libarchive/lib/pkgconfig:/usr/local
 direnv allow
 ```
 
-## Ubuntu: Latest (16.10/Yakkety)
+## Ubuntu: Latest (17.10/Artful)
 
-This installation method uses as many packages from Ubuntu as possible. This
-will closely reproduce the state of the Docker-based "devshell" as it also uses
-an Ubuntu base image.
+This installation method uses as many packages from Ubuntu as possible.
+If you don't have it, you'll first need to install `git`:
+```
+sudo -E apt-get install -y --no-install-recommends git
+```
 
-First clone the codebase and enter the directory:
+Then, clone the codebase and enter the directory:
 
 ```
 git clone https://github.com/habitat-sh/habitat.git
 cd habitat
 ```
 
-Then, run the system preparation scripts and try to compile the project:
+Then, run the system preparation scripts
 
 ```
 cp components/hab/install.sh /tmp/
 sh support/linux/install_dev_0_ubuntu_latest.sh
 sh support/linux/install_dev_9_linux.sh
-. ~/.profile
+```
+
+Ubuntu-Server 17.10 has an issue that prevents update from working because it
+tries to read from a CD-ROM that's almost certainly not there. If you get an
+error like this:
```

Comment:
Woah, that's a bad user experience from Canonical--assuming that they prep these Docker images

Reply:
It's not from docker, I'm running the ISO in VMWare. And yeah, it surprised me quite a bit.